static/app/components/onboarding/gettingStartedDoc/layout.tsx

```diff
@@ -53,15 +45,10 @@ export function Layout({
           <Divider />
         </Fragment>
       )}
-      {!isSelfHosted && newOrg && (
-        <ProductSelection platform={platformKey} withBottomMargin />
-      )}
-      {!isSelfHosted && !newOrg && (
-        <ProductSelectionAvailabilityHook
-          organization={organization}
-          platform={platformKey}
-        />
-      )}
+      <ProductSelectionAvailabilityHook
```

Comment:
Do we need to preserve the `withBottomMargin` or is that handled elsewhere?

Reply:
good catch!